nullaway/src/test/resources/com/uber/nullaway/testdata/android-error/AndroidxFragmentWithoutOnAttach.java

```diff
@@ -0,0 +1,32 @@
+package com.uber.myapplication;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.fragment.app.Fragment;
+
+public class AndroidxFragmentWithoutOnAttach extends Fragment {
+
+  @NonNull private Object mOnCreateInitialisedField;
```

Comment:
Since`@NonNull` is the default we generally don't write it in test code

Reply:
Ok, removed it :-) 
I just wanted to make it similar to the live code